.github/workflows/test_pr.yml

```diff
@@ -43,8 +50,19 @@ jobs:
           sudo bash cmake.sh --skip-license --exclude-subdir --prefix=/usr/local
         shell: bash
 
+      - name: Install vcpkg
+        run: |
+          git clone https://github.com/microsoft/vcpkg.git
+          cd vcpkg && ./bootstrap-vcpkg.sh
+          echo "VCPKG_ROOT=${{github.workspace}}/vcpkg" >> $GITHUB_ENV
+          echo "${{github.workspace}}/vcpkg" >> $GITHUB_PATH
+          ./vcpkg integrate install
+        shell: bash
```

Comment:
Will we recommend developers install vcpkg on their own then?

Reply:
Yes, this is the recommended approach - I plan to add installation instructions to the README file. The advantage of using a preset instead of hard-coding the toolchain is that it allows developers to install dependencies in their preferred way.